doc/lua_api.txt

```diff
@@ -2447,6 +2447,12 @@ Call these functions only at load time!
 * `minetest.register_on_item_eat(func(hp_change, replace_with_item, itemstack, user, pointed_thing))`
     * Called when an item is eaten, by `minetest.item_eat`
     * Return `true` or `itemstack` to cancel the default item eat response (i.e.: hp increase)
+* `minetest.register_on_item_pickup(func(item_luaentity, picker, time_from_last_punch, tool_capabilities, dir))`
```

Comment:
`item_luaentity` -> `luaentity`

I'm also wondering if giving the ObjectRef is a better API. Hmmm